library/src/blas2/rocblas_ger.cpp

```diff
@@ -1,10 +1,10 @@
 /* ************************************************************************
  * Copyright 2016-2019 Advanced Micro Devices, Inc.
  * ************************************************************************ */
+#include "rocblas_ger.hpp"
```

Comment:
I can't seem to find this file, am I somehow missing it or is it not added?

Reply:
Looks like it I forgot to add it, doing that now